### PR TITLE
Mavericks (Ruby v2) adjustments

### PR DESCRIPTION
**Pull request by ramiroaraujo**
Hi!
Loved your ruby template! I'm using a fork for my own workflows :)

The thing was for me that it didn't work quite well with Ruby v2 on mavericks, which makes sense since Mavericks wasn't around 5 months ago, hehe. Lucky for me, it helped me to understand a lot more about ruby, gems, environment, and specially native extensions.

I've made chances to make it work in Mavericks, though I don't know if it will work correctly on previous versions, specifically with Ruby 1.8.7.

Basically I: 
- changed require for require_relatve in the bundler setup require
- changed bundle call for /usr/bin/bundle to have the gems that use native extension to work correctly on Ruby that gets shipped with Mavericks
- changed the documentation to match both this changes

Hope you like it!
Cheers

### CODE REVIEW

**Comment by ramiroaraujo** [comment]
update and temporary close on the subject:
I had severe problems with gems with native extensions and RVM, and since I'm not very proficient on Ruby, I resolved to use the system ruby instead.

I still don't fully understand the why, but my dev environment for both 1.8 and 2.0 are working just great with rbenv, thus the /usr/bin/bundle change no longer has meaning.

on the other hand, I'll plan to keep a "mavericks" branch for mavericks workflows and "pre-mavericks" branch for older OSs. Since it's a different kind of change and approach, I will not do a PR, but checkout my fork and let me know.

Cheers!
